appeals/api/src/server/endpoints/appellant-cases/appellant-cases.controller.js

```diff
@@ -84,6 +90,31 @@ const updateAppellantCaseById = async (req, res) => {
 		}
 	}
 
+	try {
+		const recipientEmail = appeal.appellant?.email || appeal.agent?.email;
+		const recipientName = appeal.appellant?.email
+			? `${appeal.appellant?.firstName} ${appeal.appellant?.lastName}`
+			: `${appeal.agent?.firstName} ${appeal.agent?.lastName}`;
+		const siteAddress = appeal.address
+			? formatAddressSingleLine(appeal.address)
+			: 'Address not available';
+
+		if (recipientEmail) {
+			await req.notifyClient.sendEmail(config.govNotify.template.appealConfirmed, recipientEmail, {
+				recipient_name: recipientName,
+				appeal_reference_number: appeal.reference,
+				site_address: siteAddress
+			});
+		} else {
+			logger.info('No email address provided for either appellant or agent; no email sent.');
+		}
```

Comment:
I think we should return a status error here? If we were expecting to send an email but didn't have the required details.